spec/presenters/school_vacancies_presenter_spec.rb

```diff
@@ -18,7 +19,7 @@
 
   describe 'initialize' do
     it 'raises an error if the type is not recognised' do
-      expect { SchoolVacanciesPresenter.new(school, 'something') }.to raise_error(ArgumentError)
+      expect { SchoolVacanciesPresenter.new(school, sort, 'something') }.to raise_error(ArgumentError)
```

Comment:
Can you add some tests for the sorting behavour in this spec file too please?

Reply:
Tests added.